app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Log in</h2>
+<%= stylesheet_link_tag "style.css", "data-turbo-track": "reload" %>
+<div class="form-wrapper">
+
```

Comment:
- [ ] Kindly improve the layout of the sign in screen so that the form elements are larger and stretch across the page based on the required wireframe. The image below is what you have now.

  ![mjff](https://github.com/c00p75/blog-rails/assets/81468675/3ae678cb-47ec-45b1-a35f-a5a4b4b23aaf)